src/r5py/r5/transport_network.py

```diff
@@ -62,6 +64,47 @@ def __init__(self, osm_pbf, gtfs=[], build_config={}):
         )
         self._transport_network.transitLayer.buildDistanceTables(None)
 
+    @classmethod
+    def from_directory(cls, path):
+        """
+        Load a transport network, find input data in `path`.
+
+        This mimicks r5r’s behaviour to accept a directory path
+        as the only input to `setup_r5()`.
+
+        If more than one OpenStreetMap extract (`.osm.pbf`) is
```

Comment:
In addition, to this documentation, I think it would be good to at least warn a user if more than one PBF files were detected.